frontend/src/components/CategoryList/CategoryList.tsx

```diff
@@ -33,7 +32,7 @@ function CategoryList({ keyword }: CategoryListProps) {
   const { accessToken } = useRecoilValue(userState);
 
   const {
-    isLoading: isCategoriesLoading,
+    error: categoriesError,
```

Comment:
```suggestion
    error: categoriesGetError,
```
어떤 요청인지도 명시하는 게 어떨까요~!